### ISSUE TITLE
Old version of Chromium, new Electron session API

### ISSUE THREAD

**Issue by davidferguson** [electron/electron#6421]
I need to be able to use a Unity3dPlayer plugin within my Electron app, but have access to fairly recent Electron APIs.

In order for the Unity3dPlayer plugin to load, I saw on https://discuss.atom.io/t/embedding-a-unity3d-webplayer/24321/3 that I should use Electron 0.31.2. However, I also need access to the [Electron session API](https://github.com/electron/electron/blob/master/docs/api/session.md), which isn't available with Electron 0.31.2 as far as I can see.

Basically I need a way of running the version of Chromium that is in Electron 0.31.2, combined with the session API that's in the new Electron versions. Is this at all possible?

**Comment by zcbenz** [comment]
No, Electron's code base is very tightly bound to the Chromium version, every Chromium upgrade involves updating hundreds of lines in Electron, so there is no way to use new Electron API with old Chromium versions.
